src/mmw/apps/user/templates/user/hydroshare-auth.html

```diff
@@ -0,0 +1,32 @@
+{% include 'head.html' %}
+{% load staticfiles %}
+
+<body>
+    <div style="display:flex; flex-direction: column; height: 100%;
+                justify-content: center; align-items: center;">
+        {% if error %}
+            <div><h1>
+                <i class="fa fa-times-circle"
+                   style="font-size: 48px; margin-bottom: 24px; color: maroon"></i>
+            </h1></div>
+            <div><h1>Error</h1></div>
+            <div><p>{{ error }}</p></div>
+        {% else %}
+            <div><h1>
+                <i class="fa fa-check-circle"
+                   style="font-size: 48px; margin-bottom: 24px; color: teal"></i>
+            </h1></div>
+            <div><h1>Success!</h1></div>
+        {% endif %}
+    </div>
+    <script>
+        document.addEventListener('DOMContentLoaded', function() {
+            {% if error %}
+            var message = 'mmw-hydroshare-error';
```

Comment:
Good catch, this should be `mmw-hydroshare-failure` not `-error`. They are used here: https://github.com/WikiWatershed/model-my-watershed/blob/tt/hydroshare-export-login-frontend/src/mmw/js/src/core/modals/views.js#L292-L293. I should probably put them in constants to guard against such a mismatch.

The reason I went with strings instead of a promise is because there are three states: success, failure (during authorization), and dismissal (user canceling before authorization can complete). I hadn't thought of the case when the user cancels the authorization. I'll see if this can be refactored in to a promise ...